tests/Fixtures/FakeBucket.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\Tests\Fixtures;
+
+use WMDE\Fundraising\Frontend\BucketTesting\Bucket;
+use WMDE\Fundraising\Frontend\BucketTesting\Campaign;
+
+/**
+ * @license GNU GPL v2+
+ */
+class FakeBucket {
+
+	public static function createBucket(
+		string $bucketName = 'test',
+		bool $isDefault = true,
+		$campaignName = 'test_campaign',
+		$campaignUrlKey = 'test_url_key'
+	): Bucket {
```

Comment:
The formatting looks a bit off. I don't know if it's possible with the code standards, I prefer to fit as much parameters in one line and then continue the params on the next line at the start of the bracket. 

But it's totally up to you and I fear it's not consistent in the code base ...